docs/concepts.md

```diff
@@ -1,15 +1,98 @@
-# TriggerMesh concepts
+The following diagram depicts the core concepts of the TriggerMesh Cloud-Native Integration Platform: Source, Target, Function, Transformation, Filter, Splitter.
 
-## Source
+![](./assets/images/concepts.png)
+
+TriggerMesh allows you to integrate applications (on-premises or cloud services) by defining event sources and targets which are seen as the start and the end of a so-called Bridge. While events flow through the Bridge they can undergo splitting, filtering and transformation. Transformation of events can be done declaratively or through a function.
+
+To route events TriggerMesh makes use of the Knative eventing primitives (e.g Brokers, Channels, Triggers).
+
+Each component of a Bridge can be defined using an API object available through the TriggerMesh platform or through Knative.
```

Comment:
Suggestion: we don't mention that the Bridges' topology can be rearranged, so newcomers might believe that what's depicted in the diagram is the only possible one.

Not sure where to add that, but to me it feels like a short mention of this could be beneficial to this page.